src/Symfony/Bridge/Doctrine/ContainerAwareEventManager.php

```diff
@@ -107,7 +99,7 @@ public function hasListeners($event): bool
         return isset($this->listeners[$event]) && $this->listeners[$event];
     }
 
-    public function addEventListener($events, $listener): void
+    public function addEventListener($events, object|string $listener): void
```

Comment:
Now that you've dropeed EventManager 1, you can match the parameter types from upstream.

```suggestion
    public function addEventListener(string|array $events, object|string $listener): void
```

Please check the other overridden methods too.

Reply:
That was a question I had in mind: should I add type to everything in this class?

Well I have my answer :smile: